tests/test_users/test_schemas.py

```diff
@@ -1,24 +1,23 @@
 import copy
 from typing import Any
 from datetime import datetime, timedelta
+from contextlib import nullcontext as does_not_raise
 
 import pytest
 from pydantic.error_wrappers import ValidationError
 
 from src.apps.user.schemas import UserRegisterSchema
-from src.core.factories import UserFactory
+from src.core.factories import UserRegisterSchemaFactory
 
-@pytest.mark.parametrize('incorrect_password', 'passwordX')
-def test_passwords_are_not_identical(incorrect_password):
-    with pytest.raises(ValidationError):
-        user_data = UserFactory.build(password="password", password_repeat=incorrect_password)
-        UserRegisterSchema(**user_data)
+@pytest.mark.parametrize('password1, password2, result', [('testtest1', 'testtest1', does_not_raise()), ('testtest1', 'testtest2', pytest.raises(ValidationError))])
+def test_passwords_are_not_identical(password1, password2, result):
```

Comment:
Nazwa testu troche nie oddaje tego co on sprawdza. Taka nazwa by byla gdybys porownywal wartosci zwrocone z jakiejs funkcji, a nie validatora, ktory rzuca wyjatek. Nazwa testu powinna ten fakt w sobie zawierac, tak zeby jak test sie wywali, to po nazwie mozna od razu poznac co sie popsulo. Cos typu `test_user_register_schema_raises_validation_error_when_passwords_are_not_identical`

Reply:
I jeszcze formater kodu, bo ta linijka jest zdecydowanie za dluga